packages/thumbprint-tokens/src/__snapshots__/index.test.js.snap

```diff
@@ -362,3 +362,105 @@ $tp-wrap__no-pad-width: 1010px;
 $tp-z-index__modal: 200;
 "
 `;
+
+exports[`outputted iOS matches snapshot 1`] = `
+"import UIKit.UIColor
+
+/// These are the colors we use at Thumbtack.
+public enum Color {
+    /// Maps to #eaf6fa
```

Comment:
nit: Can we preserve the comment structure currently used in the iOS repo? (e.g., `Blue 100 – #eaf6fa.`) Just a little more formal